requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask
```

Comment:
Good practice to specify version numbers here to ensure that:

1. The python environment is consistent among deployments
2. No future updates to required dependencies break anything